Makefile

```diff
@@ -117,44 +117,45 @@ build-collector-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/collector/collector-linux ./cmd/collector/main.go
 
 .PHONY: docker
-docker: build_ui build-agent-linux build-collector-linux build-query-linux docker-images-only
+docker: build_ui build-agent-linux build-collector-linux build-query-linux build-crossdock-linux docker-images-only
 
 .PHONY: docker-images-only
 docker-images-only:
-	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
 	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema:${DOCKER_TAG} plugin/storage/cassandra/
 	@echo "Finished building jaeger-cassandra-schema =============="
+	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
```

Comment:
It's easier for this command to fail than the `jaeger-cassandra-schema`, so, keeping it as the first will make it fail "faster". Or were you having problems with this order of actions?

Reply:
It makes sense, I just moved it closer to the place where it is used.